resources/helm/cava-realtime-client/values.yaml

```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 nameOverride: cava-realtime-client
 # fullnameOverride: cava-realtime-client
-path: /messages
+path: /instrument
```

Comment:
I suggest changing the endpoint to be `/feed` since it is a data feed. `/instrument` doesn't seem to make sense since filtering might be based on an instrument, stream, site, etc.

Reply:
OK changed to feed.